libs/sdk-ui-ext/src/internal/components/configurationPanels/ConfigurationPanelContent.tsx

```diff
@@ -75,7 +81,11 @@ export default abstract class ConfigurationPanelContent<
 
     protected abstract renderConfigurationPanel(): React.ReactNode;
 
-    protected isControlDisabled(): boolean {
+    /**
+     * Optionally pass a sectionName to specialize disabling logic for specific sections.
+     * Default logic disables if no measures, error, or loading.
+     */
+    protected isControlDisabled(_sectionName?: SectionName): boolean {
```

Comment:
You probably do not need to bind the function in the constructor if you define it as an arrow function
```suggestion
    protected isControlDisabled = (_sectionName?: SectionName): boolean => {
```

then the whole constructor can simply be removed